spec/presenters/two_factor_authentication/sms_selection_presenter_spec.rb

```diff
@@ -30,7 +30,7 @@
   describe '#info' do
     context 'when a user has a phone configuration' do
       let(:phone) { build(:phone_configuration, phone: '+1 888 867-5309') }
-
+      let(:user) {}
```

Comment:
We could probably move this up to the top scope so that we only have to define it once (after line 3).

I'd also suggest making it create an actual value, since this is effectively setting `user` to `nil`, which we're not expecting to happen in any real-world usage.

```rb
let(:user) { build(:user) }
```

Same note would apply to other changes where we define `let(:user) {}` this way.